python/dgl/utils/checks.py

```diff
@@ -63,6 +63,34 @@ def prepare_tensor_dict(g, data, name):
     return {key : prepare_tensor(g, val, '{}["{}"]'.format(name, key))
             for key, val in data.items()}
 
+def parse_edges_arg_to_eid(g, edges, etid, argname='edges'):
+    """Parse edges argument and return edge IDs.
```

Comment:
Does this function return the edge ids with dtype: g.dtype and device: g.device of given edges?

Reply:
Yes and have clarified this in the docstring.